doc/encryption.md

```diff
@@ -67,7 +67,8 @@ podspec.
 ```objc
 #import "AppDelegate.h"
 
-#import <CloudantSync.h>
+#import <CloudantSync.h> // Common classes and methods
+#import <CloudantSyncEncryption.h> // Encryption classes and methods
```

Comment:
I think you should remove and/or move down these comments (line 70 and 71) to line 129-133 where you explain this import.